source/binary.js

```diff
@@ -11,7 +11,7 @@ import nAry from './nAry';
  * @memberOf R
  * @since v0.2.0
  * @category Function
- * @sig (* -> c) -> (a, b -> c)
+ * @sig (* -> c) -> ((a, b) -> c)
```

Comment:
I think this is a good improvement. Would this be better?

(a -> b -> c -> ... -> z) -> ((a, b) -> z)

Reply:
rewrite `binary` & `unary`'s signatures according to your suggestion. @Bradcomp